src/status_im/contexts/profile/contact/actions/view.cljs

```diff
@@ -22,6 +22,13 @@
                                                         :text (i18n/label :t/nickname-removed)}])
                                          (rf/dispatch [:contacts/update-nickname public-key ""]))
                                        [public-key])
+        on-show-qr                    (rn/use-callback
+                                       (fn []
+                                         (rf/dispatch [:universal-links/generate-profile-url
+                                                       {:public-key public-key
+                                                        :cb         (#(rf/dispatch [:open-modal
```

Comment:
unrelated but it might be better to update this `:cb` to something more meaningful in that event - e.g on-success etc

Reply:
brackets looks a bit off, you are actually calling dispatch here:

```
 {:public-key public-key
                                                        :cb         (#(rf/dispatch [:open-modal..))
```
That's equivalent to

```
{:public-key public-key
 :cb (rf/dispatch [:open-modal]} // note you aer actually dispatching here
```

or you want:

```
{:public-key public-key
  :cb #(rf/dispatch...)}

```
?